docs/apps/interchain-accounts/client.md

```diff
@@ -0,0 +1,232 @@
+<!--
+order: 7
+-->
+
+# Client
+
+## CLI
+
+A user can query and interact with the Interchain Accounts module using the CLI.
+
+### Controller
+
+A user can query and interact with the controller submodule.
+
+#### Query
+
+The `query` commands allow users to query the controller submodule.
+
+```shell
+simd query interchain-accounts controller --help
+```
+
+#### `interchain-account`
+
+The `interchain-account` command allows users to query the controller submodule for the interchain account address for a given owner on a particular connection.
+
+```shell
+simd query interchain-accounts controller interchain-account [owner] [connection-id]
+```
+
+Example:
+
+```shell
+simd query interchain-accounts controller interchain-account cosmos1.. connection-0
+```
+
+#### `params`
+
+The `params` command allows users to query the current controller submodule parameters.
+
+```shell
+simd query interchain-accounts controller params
+```
+
+#### Transactions
+
+The `tx` commands allow users to interact with the controller submodule.
+
+```shell
+simd tx interchain-accounts controller --help
+```
+
+##### `register`
+
+The `register` command allows users to register an interchain account on the provided connection.
+
+```
+simd tx interchain-accounts controller register [connection-id]
```

Comment:
looks like this one is missing the shell directive 

```suggestion
```shell
simd tx interchain-accounts controller register [connection-id]
```